### PR TITLE
Update CapacityExceededException handling

### PR DESCRIPTION
**Pull request by byronlin13**
*Issue #, if available:*

*Description of changes:*
-`CapacityExceededException` is a subtype of `AmazonServiceException` and contains a 503 status code which we already handle. Removed the explicit catch of `CapacityExceededException` and updated unit test to create `CapacityExceededException` with 503 status code.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: Amazon.QLDB.Driver/driver/QldbDriverBuilder.cs [hunk 1]
```diff
@@ -188,7 +188,7 @@ internal static IRetryHandler CreateDefaultRetryHandler(ILogger logger)
         private static void SetUserAgent(object sender, RequestEventArgs eventArgs)
         {
             const string UserAgentHeader = "User-Agent";
-            const string Version = "1.1.0";
+            const string Version = "1.1.1";
```
Comment (yoroto) [inline Amazon.QLDB.Driver/driver/QldbDriverBuilder.cs:191]: All other looks good. But can you move the revision update to a separate PR?

Comment (byronlin13) [reply Amazon.QLDB.Driver/driver/QldbDriverBuilder.cs]: Done

**Review by yoroto — APPROVED** [review]
